File: Frontend-Ventanilla/my-app/src/app/model/DetalleCompraClienteDTO.ts
export interface DetalleCompraClienteDTO{
    codigo: number;
    nombreProducto: string;
    descripcion: string;
    precio: number;
    cantidad: number;
    fechaVenta: string;
    horaDeVenta: string;
    direccion: string;
    devueltas: number;
    nombreCliente: string;
    correoCliente: string;
    telefono: string;
    venta: number;
}

File: Backend-Ventanilla/src/main/java/co/ventanilla_gimli/dto/ClienteDTO/DetalleCompraClienteDTO.java
package co.ventanilla_gimli.dto.ClienteDTO;

import java.time.LocalDate;

public record DetalleCompraClienteDTO(
        int codigo,
        String nombreProducto,
        String descripcion,
        double precio,
        int cantidad,
        LocalDate fechaVenta,
        String horaDeVenta,
        String direccion,
        double devueltas,
        String nombreCliente,
        String correoCliente,
        String telefono,
        int  venta
) {
}
